client/src/services/ServiceWorker/Controllers/AccountController.js

```diff
@@ -106,6 +106,18 @@ class AccountController {
 		return this.userService.addWalletAccount(index, description);
 	};
 
+	generateWallets = async ({ total }) => {
+		console.log('total: ', total);
+		if (total === 0) {
```

Comment:
Commonly we don't need this kind of data checking, this job should leave up to service handler. See `AccountController.generateKeypair` for instance. Better wrap this around a `try .. catch` block